e2e/content-services/document-list/document-list-component.e2e.ts

```diff
@@ -19,9 +19,9 @@ import { ContentServicesPage } from '../../core/pages/content-services.page';
 import { browser } from 'protractor';
 import { createApiService, LoginPage, StringUtil, UploadActions, UsersActions, ViewerPage } from '@alfresco/adf-testing';
 import { FileModel } from '../../models/ACS/file.model';
-import * as moment from 'moment';
 import { NavigationBarPage } from '../../core/pages/navigation-bar.page';
 import { NodeEntry } from '@alfresco/js-api';
+import { DateFnsUtils } from '../../../lib/core/src/lib/common/utils/date-fns-utils';
```

Comment:
incorrect import, should be `@alfresco/adf-core`

Reply:
I have updated import `@alfresco/adf-core` but the e2es are failing due to this change.

![Screenshot from 2023-10-04 19-10-24](https://github.com/Alfresco/alfresco-ng2-components/assets/112386818/8a170bb4-1023-4476-8fcc-0ea8466513bf)